clustersConfig.py

```diff
@@ -89,8 +90,10 @@ def __init__(self, yamlPath: str):
                 cc["version"] = "4.12.0-multi"
             if "external_port" not in cc:
                 cc["external_port"] = "auto"
-            if "network_api_port" not in cc:
+            if ("network_api_port" not in cc) or (cc["network_api_port"] == "auto"):
```

Comment:
styling: Don't put parenthesis in `() or ()`. There is no ambiguity.

Reply:
Done.